features/signup.feature

```diff
@@ -0,0 +1,14 @@
+Feature: We are signing up
+
+As a user
+    In order to get top my mailbox
+    I would like to login to the app
```

Comment:
Please revisit the language on this story. Not up to standard. Also, please mind the formatting.

Reply:
Please redo the whole thing. This is a feature about signing up to the system, not logging in.